plugins/auto-sizes/readme.txt

```diff
@@ -52,6 +52,17 @@ Contributions are always welcome! Learn more about how to get involved in the [C
 
 == Changelog ==
 
+= 1.2.0 =
+
+**Enhancements**
+
+* Harden logic to add `auto` keyword to `sizes` attribute to prevent duplicate keyword. ([1445](https://github.com/WordPress/performance/pull/1445))
+* Use more robust HTML Tag Processor for auto sizes injection. ([1471](https://github.com/WordPress/performance/pull/1471))
+
+**Bug Fixes**
+
+* Remove sizes attribute when the responsive image disabled. ([1399](https://github.com/WordPress/performance/pull/1399))
```

Comment:
I think there's a word missing here. Let's fix this PR's title, plus here :)

Reply:
Good catch. Fixed in fe2f03c